src/editor.js

```diff
@@ -44,16 +38,28 @@ CKEDITOR.define( [
 			 * global configurations available in {@link CKEDITOR.config} if configurations are not found in the
 			 * instance itself.
 			 *
-			 * @type {Config}
+			 * @readonly
+			 * @type {Config} config
 			 */
-			this.config = new EditorConfig( config );
+			this.set( 'config', new EditorConfig() );
 
 			/**
 			 * The plugins loaded and in use by this editor instance.
 			 *
-			 * @type {PluginCollection}
+			 * @readonly
+			 * @type {PluginCollection} plugins
 			 */
-			this.plugins = new PluginCollection( this );
+			this.set( 'plugins', new PluginCollection( this ) );
+
+			/**
+			 * The editables attached to this editor instance.
+			 *
+			 * @readonly
+			 * @type {EditableCollection} editables
+			 */
+			this.set( 'editables', new EditableCollection() );
+
+			this._creators = {};
```

Comment:
I think about `CKEDITOR` as a factory for `editors` (since it has a `create` method). And see no point in putting whole list of creators to each instance of `editor` when only one will be used. Also I think that user should not be forced to add the creator as a plugin every time he want to create an editor. It should be already defined when he build package what type of the creator will be used.